tests/e2e/api/run-newman-tests.sh

```
@@ -0,0 +1,399 @@
+# Bifrost V1 API Newman Test Runner
+# This script runs the complete Bifrost V1 API test suite using Newman
+
+set -e
+
+# Run from script directory so paths to collection and provider-capabilities.json work
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPT_DIR"
+
+# Configuration
+COLLECTION="bifrost-v1-complete.postman_collection.json"
+ENVIRONMENT="bifrost-v1.postman_environment.json"
```

Shellcheck (0.11.0)

[warning] ENVIRONMENT appears unused. Verify use (or export if used externally).

(SC2034)